app/javascript/apps/Projects/__tests__/index.test.tsx

```diff
@@ -24,26 +26,43 @@ describe("ProjectsIndex", () => {
     expect(component.find(NavBar).first().text()).toMatch("Teams")
   })
 
-  it("displays unreleased projects", () => {
-    expect(component.text()).toMatch("Out of sync")
-    expect(
-      component.find(ProjectSummaryGrid).first().props().projects.length,
-    ).toBe(1)
-    expect(
-      component.find(ProjectSummaryGrid).first().props().projects[0]
-        .isFullyReleased,
-    ).toBeFalsy()
-    expect(component.text()).toMatch("1 commit behind")
+  it("can toggle between views", () => {
+    expect(component.find(ProjectSummaryGrid).first()).toBeTruthy()
```

Comment:
I like to use the `toHaveLength` matcher for this:

```suggestion
    expect(component.find(ProjectSummaryGrid)).toHaveLength(1)
```

Docs here: https://jestjs.io/docs/en/expect#tohavelengthnumber